build-files.txt

```diff
@@ -7,6 +7,12 @@ source/dub/dub.d
 source/dub/init.d
 source/dub/packagemanager.d
 source/dub/packagesupplier.d
+source/dub/packagesuppliers/package.d
+source/dub/packagesuppliers/fallback.d
+source/dub/packagesuppliers/filesystem.d
+source/dub/packagesuppliers/packagesupplier.d
+source/dub/packagesuppliers/mvn.d
```

Comment:
I think this is really the last issue, but I'd really write that out as "maven", so that the contained class name is always the concatenation of module name and "PackageSupplier". There is also no real gain in saving characters here.

Reply:
Sure, I just used `mvn` because that's the name of the CLI tool. Renamed to "maven".